addons/Animation/src/presenter.js

```diff
@@ -60,6 +64,49 @@ function AddonAnimation_create (){
         return upgradedModel;
     };
 
+    presenter.addTextToSpeech = function (model) {
```

Comment:
Funkcje upgradujące powinny mieć w sobie nazwę upgrade. TO wygląda jak metoda, z której możnakorzystac.